src/components/AudioPlayer/PlaybackControls.tsx

```diff
@@ -1,32 +1,30 @@
 import { useSelector } from 'react-redux';
 
 import ForwardIcon from '../../../public/icons/forward_10.svg';
-import RepeatIcon from '../../../public/icons/ic_repeat_24px 1.svg';
 import ReplayIcon from '../../../public/icons/replay_10.svg';
 
 import CloseButton from './Buttons/CloseButton';
 import PlayPauseButton from './Buttons/PlayPauseButton';
 import { triggerSeek } from './EventTriggers';
 import styles from './PlaybackControls.module.scss';
+import RepeatButton from './RepeatButton';
 
 import Button, { ButtonShape, ButtonVariant } from 'src/components/dls/Button/Button';
-import { AudioFileStatus, selectAudioFileStatus } from 'src/redux/slices/AudioPlayer/state';
+import {
+  AudioFileStatus,
+  selectAudioFile,
+  selectAudioFileStatus,
+} from 'src/redux/slices/AudioPlayer/state';
 import { withStopPropagation } from 'src/utils/event';
 
 const PlaybackControls = () => {
   const audioFileStatus = useSelector(selectAudioFileStatus);
+  const audioFile = useSelector(selectAudioFile);
```

Comment:
this check should happen inside of the repeat button to determine if it's going to be disabled.